go/libraries/doltcore/doltdb/root_val.go

```diff
@@ -1055,6 +1055,36 @@ func UnionTableNames(ctx context.Context, roots ...*RootValue) ([]string, error)
 	return tblNames, nil
 }
 
+type IgnoredTables struct {
```

Comment:
The additions to this file seem more related to the code in `ignore_patterns.go` than to the code in `root_val.go`. I think it would probably fit better in there, but let me know if I'm just missing something of course. You could even rename that file to `ignore.go` or something, if you wanted the filename to carve out a bigger scope to hold more of the ignore implementation.

Reply:
Done, and renamed the file.